donate/thunderbird/templates/pages/core/ways_to_give_page.html

```diff
@@ -0,0 +1,25 @@
+{% extends "pages/core/ways_to_give_page_master.html" %}
+
+{% load i18n %}
+
+{% block tb_memo_eur %}<br><b>{% trans "Memo field:" %}</b> Thunderbird{% endblock %}
+{% block tb_memo_gbp %}<br><b>{% trans "Memo field:" %}</b> Thunderbird{% endblock %}
+
+{% block check_payee %}
+    {% blocktrans with thunderbird='Thunderbird' payee='Mozilla Foundation' trimmed %}
+    Please send a check <strong>with a separate note marked “{{thunderbird}}”</strong>, and payable (payee) to “{{payee}}”.
```

Comment:
You lost some copy here. Intentional?

Reply:
I’ve replicated what we had, so that was intentional, but I’m not 100% sure we don’t actually need that mention for TB too. Checking with Will

Also this makes me realize this line is supposed to say `Please send a check with description/memo line marked “Thunderbird”`.

![image](https://user-images.githubusercontent.com/1294206/72841115-9cfb6d80-3c8d-11ea-879a-b2902a7f191d.png)

Reply:
Landed some copy edits after chatting with Will. We’re good copy-wise